blueprints/module-unification-app/files/src/resolver.js

```diff
@@ -0,0 +1,13 @@
+import Resolver from 'ember-resolver/resolvers/fallback';
```

Comment:
I'd very much like to have the default generated app *not* use the fallback resolver. IMO new apps should default to the future/optimal mode, and if you want to use a legacy addon or `app/` directory you can opt-in to the fallback.